test/cpp_extensions/open_registration_extension/torch_openreg/third_party/openreg/CMakeLists.txt

```diff
@@ -1,4 +1,11 @@
+cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
+
+project(TORCH_OPENREG CXX C)
```

Comment:
The reasons for adding these three lines to the CMakeLists file are as follows:
- Part 1: as a subdirectory of `torch_openreg`
- Part 2: as an entry point for a separate build, as described in the examples section of the README.md file.